CHANGELOG.md

```diff
@@ -0,0 +1,4 @@
+# Changelog
+
+## 0.0.1-21 (Current)
```

Comment:
Same comment as in mapbox/mapbox-vision-ios#84:

I suggest to give explicit description on what Current means. Is that "currently is under development" or "latest stable release"?